subsys/bluetooth/rpc/Kconfig

```diff
@@ -109,3 +108,74 @@ config BT_RPC_GATT_BUFFER_SIZE
 
 endif # BT_RPC_HOST
 endif # BT_RPC
+
+choice BT_STACK_SELECTION
+
+config BT_RPC_STACK
+	bool "Bluetooth over RPC Client"
+	help
+	  Bluetooth serialized stack
+
+endchoice
+
+# BLE configuration copy. It is necessary to keep configuration copy since
+# BLE header files have dependencies to it and original samples or application
+# might use it.
+if BT_RPC_STACK
+
+config BT_CENTRAL
+	bool "Central Role support"
+	select BT_OBSERVER
+	select BT_CONN
+
+config BT_PERIPHERAL
+	bool "Peripheral Role support"
+	select BT_BROADCASTER
+	select BT_CONN
+
+config BT_OBSERVER
+	bool "Observer Role support"
+
+menu "Broadcaster"
+	visible if !BT_PERIPHERAL
+
+config BT_BROADCASTER
+	bool "Broadcaster Role support"
+	default y if !BT_OBSERVER
+	help
+	  Select this for LE Broadcaster role support.
+
+endmenu
+
+config BT_CONN
+	bool
+
+config BT_MAX_CONN
+	int "Maximum number of simultaneous connections"
+	depends on BT_CONN
+	range 1 64
+	default 1
+
+if BT_CONN
+
+config BT_REMOTE_VERSION
+	bool "Enable fetching of remote version"
+	# Enable if building a Controller-only build
+	default y if BT_HCI_RAW
+
+config BT_PHY_UPDATE
+	bool "PHY Update"
+	default y
```

Comment:
Please add some help text.

All Kconfig settings goes is rendered into docs:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/CONFIG_BT_RPC.html#help

so we should really try out best to write some proper help text on what a configuration symbol will do.

Reply:
Sure. I added a help text for all Kconfig settings